src/pages/RequestCallPage.js

```diff
@@ -280,68 +297,80 @@ class RequestCallPage extends Component {
-                                    lastName={this.state.lastName}
-                                    lastNameError={PersonalDetails.getMaxCharacterError(this.state.hasLastNameError)}
-                                    onChangeFirstName={firstName => this.setState({firstName})}
-                                    onChangeLastName={lastName => this.setState({lastName})}
-                                    style={[styles.mv4]}
-                                />
-                                <TextInput
-                                    label={this.props.translate('common.phoneNumber')}
-                                    name="phone"
-                                    keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
-                                    autoCorrect={false}
-                                    value={this.state.phoneNumber}
-                                    placeholder="2109400803"
-                                    errorText={this.state.phoneNumberError}
-                                    onBlur={this.validatePhoneInput}
-                                    onChangeText={phoneNumber => this.setState({phoneNumber})}
-                                />
-                                <TextInput
-                                    label={this.props.translate('requestCallPage.extension')}
-                                    keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
-                                    autoCompleteType="off"
-                                    autoCorrect={false}
-                                    value={this.state.phoneExtension}
-                                    placeholder="100"
-                                    errorText={this.state.phoneExtensionError}
-                                    onBlur={this.validatePhoneExtensionInput}
-                                    onChangeText={phoneExtension => this.setState({phoneExtension})}
-                                    containerStyles={[styles.mt4]}
-                                />
-                                <Text style={[styles.textMicroSupporting, styles.mt4]}>{this.getWaitTimeMessage()}</Text>
-                            </Section>
-                        </FormElement>
-                    </ScrollView>
-                    <FixedFooter>
-                        {isBlockedFromConcierge && (
-                            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mb3]}>
-                                <Icon src={Expensicons.Exclamation} fill={colors.yellow} />
-                                <Text style={[styles.mutedTextLabel, styles.ml2, styles.flex1]}>{this.props.translate('requestCallPage.blockedFromConcierge')}</Text>
-                            </View>
+                    {this.props.account.success
+                        ? (
+                            <RequestCallConfirmationScreen />
```

Comment:
I noticed that there isn't an animation when the confirmation page shows, it just flashes onto the screen. If we navigated to another page I think it might look nicer. It's not that big of a deal but I'm curious to hear your thoughts.

Reply:
Hmmm... but then we would stack one more view on top and then the back button on a mobile view would take you right back to the form which I don't love.

This is the way we did it for the PasswordPage, which was what I based this off on: https://github.com/Expensify/App/blob/53431af69017f6bb779c202377cfaed3cada555c/src/pages/settings/PasswordPage.js#L158-L160

I think it's sufficient TBH 🤷‍♂️

Reply:
Ok thanks, that's a good explanation. You're right that we don't want to back button to go back to the form.